hw/ip/otbn/README.md

```diff
@@ -99,16 +99,27 @@ and check the results are as expected.
 
 ### Run OT earlgrey simulation with the OTBN model, rather than the RTL design
 
-The default behaviour is that the OTBN block contains the RTL design.
-If you wish to run system-level testing against the Python-based
-instruction set simulator instead of the RTL (hereafter called the
-ISS), you need to pass the `OTBN_MODEL` flag. For example, to compile
-the Verilator simulation, use:
+For simulation targets, the OTBN block can be built with both the RTL
+implementation and the Python-based instruction set simulator (hereafter called
+the ISS) compiled in. When running the simulation the plusarg `OTBN_MODEL_USE`
```

Comment:
Do you think we could rename these to `OTBN_BUILD_MODEL` and `OTBN_USE_MODEL`? I understand that the names you've got are logical (`otbn` -> `model` -> verb), but they look a bit odd.